### ISSUE TITLE
webui: add "open file" dialog for upload

### ISSUE THREAD

**Issue by cyphar** [camlistore/camlistore#830]
Currently the only way to upload a file to the UI is to drag and drop it. Having a traditional "open file" dialog that allows you to select a set of files should be a fallback (for both accessibility and sanity reasons).

**Comment by mpl** [comment]
One more (even more important imho) reason is that one can't do the drag and drop on mobile, so there's no way to upload when using the web UI on mobile.

**Comment by mpl** [comment]
https://camlistore-review.googlesource.com/8006
